tests/_support/Entity/Cast/CastUUID.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace Tests\Support\Entity\Cast;
+
+use CodeIgniter\Entity\Cast\BaseCast;
+
+class CastUUID extends BaseCast
```

Comment:
Since this is test data it's not a big deal but more accurate would be `CastBinaryUUID`